Week2/num6.js

```diff
@@ -0,0 +1,15 @@
+// 6 Arrays
+// 6.1
+let Friends = [];
```

Comment:
you don't have to use uppercase `F` here. If you start the variable name with a capital letter, it might suggest that it is a `class`.

So `let friends = [];` is better (see [naming conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md))